src/main/java/fm/last/moji/impl/MojiImpl.java

```diff
@@ -72,7 +72,7 @@ public void copyToMogile(File source, MojiFile destination) throws IOException {
       outputStream.flush();
     } finally {
       IOUtils.closeQuietly(inputStream);
-      IOUtils.closeQuietly(outputStream);
+      outputStream.close();;
```

Comment:
I find this surprising behaviour as line 72 clearly flushes the buffer. Are you certain this is where the problem lies? How did you find it?